docs/references/architecture/recipes/about-recipes.md

```diff
@@ -7,4 +7,4 @@ draft: true
 
 Tauri recipes are descriptions of use-cases that are entirely configurable within the `src-tauri/tauri.conf.json` file. These are not the limits of what Tauri can do, and there are probably more out there. If you discover one, why not get in touch and help us update this collection!
 
-If you haven't read about the general design of Tauri, then it would make the most sense for you to visit the ["Getting Started"](../../getting-started/beginning-tutorial) and become familiar with the basic architecture and terminology used in these recipes.
+If you haven't read about the general design of Tauri, then it would make the most sense for you to visit the ["Quick Start"](../../../guides/getting-started/setup/) and become familiar with the basic architecture and terminology used in these recipes.
```

Comment:
If you could update this to use reference-style links that would be brilliant! You can find some more details in the first point here: https://github.com/tauri-apps/tauri-docs/blob/dev/.github/CONTRIBUTING.md#style

For links going to another internal (markdown) page, the link should end in the `.md` file extension so that broken-link detection is build-in (so it would be `../../../guides/getting-started/setup.md` instead of `../../../guides/getting-started/setup/`)

Reply:
Ok, done